tests/dialects/test_clickhouse.py

```diff
@@ -45,6 +45,9 @@ def test_clickhouse(self):
         self.validate_identity(
             "CREATE MATERIALIZED VIEW test_view ON CLUSTER cl1 (id UInt8) ENGINE=AggregatingMergeTree() ORDER BY tuple() AS SELECT * FROM test_data"
         )
+        self.validate_identity(
+            "CREATE MATERIALIZED VIEW test_view ON CLUSTER cl1 (id UInt8) TO table1 AS SELECT * FROM test_data"
+        )
```

Comment:
From the docs it seems like `db.table` is also a valid form (see: `[TO[db.]name]`), should we add a test for it too?

Reply:
Good idea!